brambling/templates/brambling/event/organizer/update.html

```diff
@@ -1,9 +1,295 @@
+{% load floppyforms %}
```

Comment:
If I felt confident that I knew how to write integration tests in this framework, I'd totally write ones to make a request to each view and assert that it gives a 200 response during a Dancerfly sprint.